main/src/library/Abort.flix

```diff
@@ -24,14 +24,21 @@ pub eff Abort {
     ///
     /// The computation cannot be resumed.
     ///
-    def abort(m: RichString): Void
+    def abortWithMsg(m: RichString): Void
 
 }
 
 mod Abort {
 
     import java.lang.System
 
+    ///
+    /// Immediately aborts the current computation with the given error message `m`.
+    ///
+    /// The computation cannot be resumed.
+    ///
+    pub def abort(m: a): Void \ Abort + Formattable.Aef[a] with Formattable[a] = Abort.abortWithMsg(Formattable.format(m))
```

Comment:
Should this return `b` instead?

Reply:
Hmm. Maybe? This PR is kinda broken anyway.